android/src/main/java/vn/luongvo/kmm/survey/android/ui/common/NextCircleButton.kt

```diff
@@ -0,0 +1,46 @@
+package vn.luongvo.kmm.survey.android.ui.common
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.Button
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color.Companion.White
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import vn.luongvo.kmm.survey.android.R
+import vn.luongvo.kmm.survey.android.ui.theme.AppTheme.dimensions
+
+@Composable
+fun NextCircleButton(
```

Comment:
@luongvo Not a big due, just optional. As it is a common button, what do you think if we name this like `CircleButton` with the drawable id parameter?
I understand that we do not reuse with other drawable so far. We can do the refactor later if need.

Reply:
@doannimble Yep, valid point. I will consider refactoring if needed to expand it to be more common with a drawableId param 👍